data/sql/updates/pending_db_world/rev_1546538959760274673.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1546538959760274673');
+
+ALTER TABLE `game_event`
+ADD COLUMN `announce` tinyint(3) unsigned NULL DEFAULT 2 COMMENT '0 dont announce, 1 announce, 2 value from config' AFTER `world_event`;
```

Comment:
what does the `NULL` after `unsiged` do?

Reply:
i miss not null >_<